Microsoft.Toolkit.Services/Microsoft.Toolkit.Services.csproj

```diff
@@ -30,7 +30,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
+    <PackageReference Include="System.Text.Json" Version="4.7.2" />
```

Comment:
@Sergio0694 we can multi-target to filter this out for .NET Standard 2.1, right?

Reply:
Unfortunately `System.Text.Json` isn't part of .NET Standard 2.1, so we can't 😟

Reply:
Ah, so it's just part of .NET 5, so we have to wait until we target that specifically then.